src/nrnpython/grids.h

```diff
@@ -10,6 +10,7 @@ and Flux_pair structs and their respective functions
 
 #include <nrnwrap_Python.h>
 
+#define PYHOC_REF_TO_NUMBER 4
```

Comment:
Instead of doing this, is it practical to move the definition of [PyHoc::ObjectType](https://github.com/neuronsimulator/nrn/blob/99a6a370ef143b3c681eda1deea68bc7e2b8f21d/src/nrnpython/nrnpy_hoc.cpp#L159 ) into a header file? (I guess the most obvious choice is `nrnpy_hoc_2.h` and `nrnpy_hoc_3.h`, but I hate to see this duplicated so maybe `nrnpython.h`?)